modules/python/test/test_misc.py

```diff
@@ -265,7 +265,7 @@ def test_parse_to_int_not_convertible(self):
     def test_parse_to_int_not_convertible_extra(self):
         for not_convertible in (np.bool_(True), True, False, np.float32(2.3),
                                 np.array([3, ], dtype=int), np.array([-2, ], dtype=np.int32),
-                                np.array([1, ], dtype=np.int), np.array([11, ], dtype=np.uint8)):
+                                np.array([11, ], dtype=np.uint8)):
```

Comment:
`np.uint8`

Reply:
I removed np.array([1, ], dtype=np.int) because it is np.array([1, ], dtype=int), and there is already a np.array([3, ], dtype=int) above.